scripts/scil_tractogram_alter.py

```diff
@@ -0,0 +1,156 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+
+
+"""
+This script is used to alter a bundle to reach specific minimum dice
+coefficient from the original bundle. The script will subsample, trim, cut,
+replace or transform the streamlines until the minimum dice is reached.
+(cannot be combined in one run, use the script multiple times if needed)
+
+All operation use a dichotomic search to find the parameter that gets as close
+as possible to the specified minimum dice coefficient (with an epsilon for
+convergence).
+
+The subsample operation will remove streamlines until the minimum dice is
+reached. This affect the whole bundle.
+
+The trim operation will used the lowest density voxels (starting at 1) to
+remove points from the streamlines until the minimum dice is reached.
+This typically affect the edge of the bundle.
+
+The cut operation will remove points from the start (or end) streamlines until
+the minimum dice is reached. This affect one end of the bundle.
+
+The replace operation will upsample the tractogram (generate new streamlines
+with noise) and then subsample the tractogram. This effectively replace
+streamlines with similar ones until the minimum dice is reached.
+This affect the whole bundle.
+
+The transform operation will apply random transformations to the streamlines
+until the minimum dice is reached. This affect the whole bundle.
+"""
```

Comment:
Please cite the other scripts. 

Ex, "For more description of the upsampling parameters, see scil_tractogram_resample"

------------

Also, can you add: the operations are performed in the following order: 1) subsampling, 2) trimming, 3) cutting, 4) upsampling